src/main/target/TASK_F405_For_Skye

```diff
@@ -0,0 +1,3 @@
+target.c
```

Comment:
/*
 * This file is part of Cleanflight.
 *
 * Cleanflight is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Cleanflight is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
 */

#include <stdint.h>

#include <platform.h>
#include "drivers/io.h"

/*#include "drivers/dma.h"*/
#include "drivers/timer.h"

const timerHardware_t timerHardware[] = {
    //DEF_TIM(TIM4, CH3, PB8,  TIM_USE_PPM,                               0, 0), // PPM

    // Motors
    DEF_TIM(TIM4,  CH3,  PB8,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR,      0, 0), // S1_OUT D2_ST6
    DEF_TIM(TIM4,  CH4,  PB9,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,      0, 0), // S2_OUT D1_ST2
    DEF_TIM(TIM4,  CH2,  PB7,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,      0, 0), // S3_OUT D1_ST6
    DEF_TIM(TIM4,  CH1,  PB6,  TIM_USE_MC_MOTOR | TIM_USE_FW_SERVO,      0, 0), // S4_OUT D1_ST1

    // LED strip
    //DEF_TIM(TIM4,  CH1, PB6,  TIM_USE_LED,                              0, 0), // D1_ST0
};

const int timerHardwareCount = sizeof(timerHardware) / sizeof(timerHardware[0]);